.circleci/config.yml

```diff
@@ -110,9 +101,42 @@ aliases:
           - master
           - /docs.+/
     requires:
-      - bootstrap
+      - bootstrap-18.2.0
+
+  integration-test-workflow: &integration-test-workflow
```

Comment:
we were using `e2e-test-workflow` for both e2e and integration, which was confusing and inflexible. I split them so I could set up the node version parameter and dependency on parameterized bootstrap version.